next/src/field/schema.ts

```diff
@@ -169,6 +173,80 @@ function getFieldOptions(schema: NonBooleanJsfSchema) {
   return null
 }
 
+/**
+ * Get the fields for an object schema
+ * @param schema - The schema of the field
+ * @param strictInputType - Whether to strictly enforce the input type
+ * @returns The fields for the schema or an empty array if the schema does not define any properties
+ */
+function getObjectFields(schema: NonBooleanJsfSchema, strictInputType?: boolean): Field[] | null {
+  const fields: Field[] = []
+
+  for (const key in schema.properties) {
+    const isRequired = schema.required?.includes(key) || false
+    const field = buildFieldSchema(schema.properties[key], key, isRequired, strictInputType)
+    if (field) {
+      fields.push(field)
+    }
+  }
+
+  const orderedFields = setCustomOrder({ fields, schema })
```

Comment:
comment: what's the rational for the function signature? In some places the function accepts multiple parameters and in another places (like this) uses an object as a parameter. 
This is not a big deal not a blocker either but if we can standardize it in the future, it would be good IMO.

Reply:
Didn't write that function but I prefer multiple params tbh and we use that everywhere else. I'll change this one to match the rest.